.github/workflows/publish.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    tags:
+      - "v*"
+
+name: Publish
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - run: cargo test --features="serde"
+      - run: cargo clippy -- -Dwarnings
+      - run: cargo publish --token ${CRATES_TOKEN}
```

Comment:
Please consider adding the `sine-fdn` as a crate owner also; if my understanding is correct, it'd be through `cargo owner --add github:sine-fdn:staff`